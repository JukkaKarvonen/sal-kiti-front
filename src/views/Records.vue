<template>
  <div class="records">
    <Records />
  </div>
</template>

<script>
/**
 * Records search view
 */
import Records from "@/components/RecordsForm.vue";

export default {
  name: "RecordsView",
  components: {
    Records
  },
  created() {
    document.title = this.$t("title.records") + " | " + this.$t("title.prefix");
  }
};
</script>
