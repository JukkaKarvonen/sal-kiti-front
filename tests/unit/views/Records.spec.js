import { shallowMount, createLocalVue } from "@vue/test-utils";
import Records from "@/views/Records.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("Records.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(Records, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
