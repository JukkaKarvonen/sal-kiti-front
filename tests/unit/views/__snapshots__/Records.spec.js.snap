// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Records.vue renders content 1`] = `
<div
  class="records"
>
  <records-stub />
</div>
`;
